git_pile/git_pile.py

```diff
@@ -315,13 +315,13 @@ def cmd_init(args):
         #
         # Workaround is to do that ourselves with a temporary repository
         with tempfile.TemporaryDirectory() as d:
-            git("-C %s init" % d)
+            git("-C %s init -b pile" % d)
```

Comment:
Will this now fail ("-b" unrecognized) on pre-2.28 git versions?  The README still says that we only need git >= 2.19.

Reply:
ugh, no. I didn't realize it wasn't available in pre-2.28. I'd rather add a little compatible layer. In this case maybe a function that first tries with -b, and if it doesn't work, a git-init + git branch.